spec/api/v1/projects_spec.rb

```diff
@@ -75,44 +75,50 @@
 
   describe '#GET /api/v1/projects/{slug}/analysis' do
     let(:project) { create :project }
-    let(:date) { Time.new(2016, 12, 7, 10, 0) }
+    let(:date) { Time.new(2016, 12, 1) }
```

Comment:
Would be nice to name this as `date_for_iteration_number`, as now there are lots of `_date` variables.